### DISCUSSION TITLE
弱弱问一句,EasyCaching这类 只存Key作为索引的缓存,一般怎么缓存复杂一点的查询之类的操作和对象呢.

### DISCUSSION THREAD

**Discussion by sgf** [dotnetcore/EasyCaching#435] [Q&A]
弱弱问一句,EasyCaching这类 只存Key作为索引的缓存,一般怎么缓存复杂一点的查询之类的操作和对象呢.

**Comment by sgf** [comment]
有的时候 不仅仅是需要Key还需要少量的其他条件.

某种意义上来说,我感觉或许我需要的是一种 数据库查询的二级缓存.

但是我找遍了github 似乎并没有好的选择.

**Comment by catcherwong** [comment]
这种需求不太行

**Reply by sgf** [reply]
> 这种需求不太行

其实一般除了主键,也就多查两三个字段, emm.
不行的话,我再找找吧. 有两个开源的 Cache是可以的,不过貌似是商用的.
